v0/destinations/ga/transform.js

```diff
@@ -299,11 +299,17 @@ function responseBuilderSimple(
       ? message.integrations[gaDisplayName].clientId
       : undefined
     : undefined;
-  finalPayload.cid =
-    integrationsClientId ||
-    getDestinationExternalID(message, "gaExternalId") ||
-    message.anonymousId ||
-    md5(message.userId);
+
+  const disableMd5 = get(message, "disableMd5");
+  if (disableMd5) {
+    finalPayload.cid = undefined;
```

Comment:
we shouldn't keep this undefined. if md5 is disabled 
cid =  finalPayload.cid =
      integrationsClientId ||
      getDestinationExternalID(message, "gaExternalId") ||
      message.anonymousId ||
     message.userId;

Reply:
Okay, got it

Reply:
changes are updated